[BUG] uuid header

when I try to build the project appear:

```bash
make
[  2%] Building C object atividades/sockets_locais_threads/CMakeFiles/client.dir/client.c.o
/home/alunoifpe/MONITORIA_SISTEMAS_DISTRIBUIDOS/atividades/sockets_locais_threads/client.c:8:10: fatal error: uuid/uuid.h: No such file or directory
    8 | #include <uuid/uuid.h>
      |          ^~~~~~~~~~~~~
compilation terminated.
make[2]: *** [atividades/sockets_locais_threads/CMakeFiles/client.dir/build.make:76: atividades/sockets_locais_threads/CMakeFiles/client.dir/client.c.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:326: atividades/sockets_locais_threads/CMakeFiles/client.dir/all] Error 2
make: *** [Makefile:91: all] Error 2
```

--- a/INSTALL_DEPS.sh
+++ b/INSTALL_DEPS.sh
@@ -20,4 +20,7 @@ apt install -y build-essential
 echo "Instalando OpenSSL e bibliotecas de desenvolvimento..."
 apt install -y libssl-dev
 
+echo "Instalando a biblioteca que gera uuid..."
+apt install -y uuid-dev
+
 echo "Dependências instaladas com sucesso!"

